server/plugins/schema-plugin-ob-mysql/src/main/java/com/oceanbase/odc/plugin/schema/obmysql/parser/OBMySQLGetDBTableByParser.java

```diff
@@ -244,33 +248,90 @@ public DBTablePartition getPartition() {
         if (partitionStmt.getSubPartitionOption() == null) {
             return partition;
         }
-        // TODO 目前 ODC 仅支持 HASH/KEY 二级分区, 其它类型后续需补充
+        fillSubPartitions(partition, partitionStmt);
+        return partition;
+    }
+
+    private void fillSubPartitions(DBTablePartition partition, Partition partitionStmt) {
         partition.setSubpartitionTemplated(partitionStmt.getSubPartitionOption().getTemplates() != null);
-        SubPartitionOption subOption = partitionStmt.getSubPartitionOption();
+        DBTablePartitionOption subPartitionOption = partition.getSubpartition().getPartitionOption();
+        List<DBTablePartitionDefinition> subPartitionDefinitions =
```

Comment:
will it be null？

Reply:
![image](https://github.com/user-attachments/assets/d49e55d9-19f2-49d8-94c6-fdfe5824bfa8)
not null